probes/packagedWithAutomatedWorkflow/impl.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:stylecheck
+package packagedWithAutomatedWorkflow
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/internal/utils/uerror"
+)
+
+//go:embed *.yml
+var fs embed.FS
+
+const Probe = "packagedWithAutomatedWorkflow"
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	if raw == nil {
+		return nil, "", fmt.Errorf("%w: raw", uerror.ErrNil)
+	}
+
+	r := raw.PackagingResults
+	for _, p := range r.Packages {
+		if p.Msg == nil {
+			// Presence of a single non-debug message means the
+			// check passes.
+			f, err := finding.NewWith(fs, Probe,
+				"Project packages its releases by way of Github Actions.", nil,
+				finding.OutcomePositive)
```

Comment:
1. we might miss other packaging workflows with an early exit right? The code previously processed all of them.
2. can we add the location here to the finding if we have it?
The code previously had this block:
```
	if p.File != nil {
		msg.Path = p.File.Path
		msg.Type = p.File.Type
		msg.Offset = p.File.Offset
	}
```

Reply:
Done in https://github.com/ossf/scorecard/pull/3486/commits/5e460a33d0adee393a6c58c9e8355b71c858303d